parts/kubernetesagentcustomdata.yml

```diff
@@ -147,6 +147,18 @@ write_files:
   content: !!binary |
     {{WrapAsVariable "provisionScript"}}
 
+- path: "/opt/azure/containers/useoverlay2.sh"
```

Comment:
why do you need this script?  You already set this file above.

Reply:
Oh yes, it's not needed. Sorry for the mixup. Will fix it in a bit.